### PR TITLE
Bump dependencies and version for GA

### PR DESCRIPTION
**Pull request by davidwengier**
Updating dependencies and version numbers to public packages.

@JoeRobich is this targetting the right branch? or should it target `release/dev17.0`

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
+    <PatchVersion>1</PatchVersion>
     <PreReleaseVersionLabel>6</PreReleaseVersionLabel>
```
Comment (davidwengier) [inline eng/Versions.props:11]: Should this be reset to 0?

Comment (JoeRobich) [inline eng/Versions.props:11]:
```suggestion
    <PreReleaseVersionLabel>1</PreReleaseVersionLabel>
```

**Comment by JoeRobich** [comment]
> @JoeRobich is this targetting the right branch? or should it target `release/dev17.0`

Targeting this branch is fine. We can delete the dev17.0 branch now that we have GA.

**Review by JoeRobich — APPROVED** [review]

**Pull request merged**